op-chain-ops/cmd/check-delta/check-delta.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash
set -euo pipefail
```

will ensure the script fails if one of the command fails rather than silently ignoring errors.

Reply:
`e` - fail on error
`u` - fail on undeclared variables (picks up a lot of typos)
`o pipefail` - makes bashes handling of exit codes as part of a pipe chain sane.

Reply:
Applied suggestion at https://github.com/ethereum-optimism/optimism/pull/8417/commits/d43897494911165823c3f6310fa421dd2f6bf83a. Dropped `-e` option to avoid exiting when `jq` errored.